src/System.Runtime.Serialization.Formatters/tests/BinaryFormatterTests.cs

```diff
@@ -252,16 +45,87 @@ public void ValidateBasicObjectsRoundtrip(object obj, FormatterAssemblyStyle ass
             Assert.Equal(obj, result);
         }
 
+        private static object LoadFromFile(string base64Str)
+        {
+            var binaryFormatter = new BinaryFormatter();
+            byte[] serializedObj = Convert.FromBase64String(base64Str);
+            using (var serializedStream = new MemoryStream(serializedObj))
+            {
+                return binaryFormatter.Deserialize(serializedStream);
+            }
+        }
+
+        [Theory]
+        [MemberData(nameof(SerializableObjects))]
+        public void ValidateAgainstFile(object original, string coreBase64, string netfxBase64)
+        {
+            CheckForAnyEquals(original, LoadFromFile(coreBase64));
+            //CheckForAnyEquals(originalType, LoadFromFile(netfxBase64));
```

Comment:
intentional for this PR. I rushed to finish core <--> core serialization and also moved test data to code as @stephentoub suggested.